Tools/autotest/helicopter.py

```diff
@@ -697,6 +698,52 @@ def check_airspeeds(mav, m):
             self.disarm_vehicle()
             self.context_pop()
 
+    def TurbineCoolDown(self, timeout=200):
+        """Check Turbine Cool Down Feature"""
+        # set option for Turbine
+        RAMP_TIME = 4
+        SETPOINT = 66
+        IDLE = 15
+        COOLDOWN_TIME = 5
+        self.set_parameter("RC6_OPTION", 161)
+        self.set_parameter("H_RSC_RAMP_TIME", RAMP_TIME)
+        self.set_parameter("H_RSC_SETPOINT", SETPOINT)
+        self.set_parameter("H_RSC_IDLE", IDLE)
+        self.set_parameter("H_RSC_CLDWN_TIME", COOLDOWN_TIME)
+        self.set_rc(3, 1000)
+        self.set_rc(8, 1000)
+
+        self.progress("Starting turbine")
+        self.wait_ready_to_arm()
+        self.arm_vehicle()
+
+        self.set_rc(6, 2000)
+        self.wait_statustext('Turbine startup')
```

Comment:
There's a race condition here, should collect statustext in the context and `check_context=True`